tests/v2/validation/upgrade/kubernetes_test.go

```diff
@@ -106,4 +110,23 @@ func (u *UpgradeKubernetesTestSuite) testUpgradeSingleCluster(clusterName, versi
 		u.T().Logf("[%v]: Validating updated cluster's nodepools kubernetes versions", clusterName)
 		validateNodepoolVersions(u.T(), client, updatedCluster, version, !isCheckingCurrentCluster)
 	}
+
+	err = nodestat.IsNodeReady(client, clusterMeta.ID)
+	require.NoError(u.T(), err)
```

Comment:
So I wrote this comment to a resolved comment, I agree with you, is this check necessary since we already check the cluster's active state before it gets out of the polls?

Reply:
I would say so, because this checks the node's status. I've seen times while manually testing where just because the cluster is active, it doesn't guarantee the nodes are active as well.